pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1576,6 +1576,12 @@ func setDaemonSetToleration(ds *apps.DaemonSet, tolerations []v1.Toleration) {
 // DaemonSet should launch a critical pod even when the node with OutOfDisk taints.
 // TODO(#48843) OutOfDisk taints will be removed in 1.10
 func TestTaintOutOfDiskNodeDaemonLaunchesCriticalPod(t *testing.T) {
+	enabled := utilfeature.DefaultFeatureGate.Enabled(features.ExperimentalCriticalPodAnnotation)
+	defer func() {
+		utilfeature.DefaultFeatureGate.Set(fmt.Sprintf("%s=%t",
+			features.ExperimentalCriticalPodAnnotation, enabled))
```

Comment:
Check error

Reply:
oh, sorry for the miss; and done :)

Reply:
also go through the whole file and updated :)